src/components/Upload/CommonFields/OptionalAnalysis/index.jsx

```diff
@@ -20,112 +21,172 @@ import React from "react";
       </InputContainer>
+
       <InputContainer
         type="checkbox"
         checked={analysis.mime}
         name="mime"
         id="upload-analysis-mime"
-        onChange={(e) => handleChange(e)}
+        onChange={(checked) => handleChange(checked, "mime")}
       >
-        MIME-type Analysis (Determine mimetype of every file. Not needed for
-        licenses or buckets)
+        MIME-type Analysis
+        <Tooltip title="Determines the mimetype of every file, which can be used to ignore specific file types" />
       </InputContainer>
+
       <InputContainer
         type="checkbox"
         checked={analysis.monk}
         name="monk"
         id="upload-analysis-monk"
-        onChange={(e) => handleChange(e)}
+        onChange={(checked) => handleChange(checked, "monk")}
       >
-        Monk License Analysis, scanning for licenses performing a text
-        comparison
+        Monk License Analysis
+        <Tooltip title="Scanning for licenses by text comparison" />
       </InputContainer>
+
       <InputContainer
         type="checkbox"
         checked={analysis.nomos}
         name="nomos"
         id="upload-analysis-nomos"
-        onChange={(e) => handleChange(e)}
+        onChange={(checked) => handleChange(checked, "nomos")}
       >
-        Nomos License Analysis, scanning for licenses using regular expressions
+        Nomos License Analysis
+        <Tooltip title="Scanning for Licenses using regular expressions" />
       </InputContainer>
+
       <InputContainer
         type="checkbox"
         checked={analysis.ojo}
         name="ojo"
         id="upload-analysis-ojo"
-        onChange={(e) => handleChange(e)}
+        onChange={(checked) => handleChange(checked, "ojo")}
       >
-        Ojo License Analysis, scanning for licenses using
-        SPDX-License-Identifier
+        Ojo License Analysis
+        <Tooltip title="Scanning for licenses using SPDX-License-Identifier" />
       </InputContainer>
+
       <InputContainer
         type="checkbox"
         checked={analysis.package}
         name="package"
         id="upload-analysis-package"
-        onChange={(e) => handleChange(e)}
+        onChange={(checked) => handleChange(checked, "package")}
+      >
+        Package Analysis
+        <Tooltip title="Parsing package headers for files ex. if files are rpm package listed, display their package information" />
+      </InputContainer>
+      <InputContainer
+        type="checkbox"
+        checked={analysis.reuse}
+        name="reuse"
+        id="upload-analysis-reuse"
+        onChange={(checked) => handleChange(checked, "reuse")}
+      >
+        REUSE.Software Analysis
+        <Tooltip title="ReSo agent marks licensed files with a license found in the .license files (outside of the licensed files)." />
+      </InputContainer>
+      <InputContainer
+        type="checkbox"
+        checked={analysis.toolkit}
```

Comment:
Please don't call this agent as `toolkit` rather as `scanoss` as ScanCode-Toolkit is generally referred with this name and it can confuse users.